docker/local_postgres/0007_openledger_audio_view.sql

```diff
@@ -9,7 +9,8 @@ INSERT INTO public.audio_popularity_metrics (
   provider, metric, percentile
 ) VALUES
   ('wikimedia_audio', 'global_usage_count', 0.85),
-  ('jamendo', 'listens', 0.85);
+  ('jamendo', 'listens', 0.85),
+  ('freesound', 'num_downloads', 0.85);
```

Comment:
Nice catch! Perhaps`freesound` should be included in the title too?

Reply:
This is _just_ the SQL DDL, which is only used locally. Since Freesound was present in logic file, it was added (either manually or automatically) to the appropriate table quite a while ago. Confirmed in production:

```sql
deploy@localhost:openledger> select * from audio_popularity_metrics;
+-----------------+--------------------+------------+
| provider        | metric             | percentile |
|-----------------+--------------------+------------|
| jamendo         | listens            | 0.85       |
| wikimedia_audio | global_usage_count | 0.85       |
| freesound       | num_downloads      | 0.85       |
+-----------------+--------------------+------------+
SELECT 3
Time: 0.098s
```

So this is more a bookkeeping change than anything 😄